### ISSUE TITLE
Deal better with errors (missing yq)

### ISSUE THREAD

**Issue by Mortimerp9** [joshmedeski/tmux-nerd-font-window-name#36]
Hello,

when the script fails, the tmux tab name is a very long:
``` 
'/home/mortimer/.tmux/plugins/tmux-nerd-font-window-name/bin/tmux-nerd-font-window-name bash 1' not ready
```

This is quite cumbersome and not actually informative.
![Screenshot 2024-03-22 at 14 44 57](https://github.com/joshmedeski/tmux-nerd-font-window-name/assets/628467/4253a81b-b2ed-4d24-8e57-5fccb804f28d)

I ran the script manually and found out that this time it's because I am missing yq on the machine I moved to, but sometimes it's for other reasons.

It would be good to improve the error handling to fall back on some display that doesn't make the tmux window name bar unusable.

Thanks

**Comment by joshmedeski** [comment]
Do you have any suggestions for what the error messages could be? I wouldn't want to just do nothing as that could be misleading.

**Comment by Mortimerp9** [comment]
good question, I am not sure, a simple "unable to name" with an error-like icon (or just the fallback) would maybe be more active. Or maybe a fallback for `get_config_value` if `yq` is not installed. 

My use case is that I log into a lot of different machines, I can move some dotfiles/plugins like this one easily, but installing binaries is not always easy. So I can't always have yq around.

If I find some time I might send you a PR to try to parse the basic yaml config without yq.

**Comment by joshmedeski** [comment]
Thanks for the feedback, I'll come up with something and update it in the next week or so.

**Comment by james1236** [comment]
This was patched already, so its just because I was at be794ba (v2.2.0-ish). I didn't know tpm only updates plugins when you do `prefix + U`.

But since I made the patch anyway:

tmux itself generates the ["'<filepath> bash 1' not ready" error message text](https://github.com/tmux/tmux/blob/bf7bbba7eab0187ba6c127345a56e5b7bf676cf4/format.c#L447) as a timeout after [1 second of a '#()' command's execution](https://github.com/tmux/tmux/blob/bf7bbba7eab0187ba6c127345a56e5b7bf676cf4/CHANGES#L1088)

Patch:
```diff
diff --git a/tmux-nerd-font-window-name.tmux b/tmux-nerd-font-window-name.tmux
index 6ca4495..14c11df 100755
--- a/tmux-nerd-font-window-name.tmux
+++ b/tmux-nerd-font-window-name.tmux
@@ -1,4 +1,12 @@
 #!/usr/bin/env bash
 
 CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-tmux set-option -g automatic-rename-format "#($CURRENT_DIR/bin/tmux-nerd-font-window-name #{pane_current_command} #{window_panes})"
+JOB="#($CURRENT_DIR/bin/tmux-nerd-font-window-name #{pane_current_command} #{window_panes})"
+
+# replace tmux's timeout "'<filepath> bash 1' not ready" text with the current command instead
+tmux set-option -g automatic-rename-format "#{?#{m:*not ready*,$JOB},#{pane_current_command},$JOB}"
```
